packages/main/src/InputPopover.hbs

```diff
@@ -2,6 +2,7 @@
 	<ui5-responsive-popover
 		no-arrow
 		_disable-initial-focus
+		prevent-focus-restore
```

Comment:
I am adding such property on the Popover, but in another change. Do we already have it?